crates/cxx-qt-gen/src/generator/rust/mod.rs

```diff
@@ -70,7 +70,11 @@ impl GeneratedRustBlocks {
         Ok(GeneratedRustBlocks {
             cxx_mod: parser.passthrough_module.clone(),
             cxx_mod_contents,
-            namespace: parser.cxx_qt_data.namespace.clone(),
+            namespace: parser
+                .cxx_qt_data
+                .namespace
+                .clone()
+                .unwrap_or_else(|| "".to_owned()),
```

Comment:
is `unwrap_or_default()` any better here? guess is it obvious enough it's a string